src/inspector_io.cc

```diff
@@ -236,6 +236,7 @@ void InspectorIo::WriteCbIO(uv_async_t* async) {
 template<typename Transport>
 void InspectorIo::WorkerRunIO() {
   uv_loop_t loop;
+  loop.data = nullptr;
```

Comment:
I don't think Coverity is aware of the fact...

"CID 166789 (#1 of 1): Uninitialized pointer read (UNINIT)
2. uninit_use_in_call: Using uninitialized value loop.data when calling uv_loop_init."